usage.go

```diff
@@ -95,6 +95,13 @@ func (p *Parser) WriteHelp(w io.Writer) {
 				}
 				fmt.Fprint(w, spec.help)
 			}
+			// Check if spec.dest is zero value or not
+			// If it isn't a default value have been added
+			v := spec.dest
+			z := reflect.Zero(v.Type())
+			if v.Interface() != z.Interface() {
+				fmt.Fprintf(w, " [default: %s]", v)
```

Comment:
Changed this to use %v instead. %q did not work for int (I added an int value to the usage test to verify that it works for non string values) it gave '*'